web/src/components/tasks/business-formation/BusinessFormationPaginator.test.tsx

```diff
@@ -52,6 +52,7 @@ import {
 import { Business } from "@businessnjgovnavigator/shared/userData";
 import * as materialUi from "@mui/material";
```

Comment:
So there are a few other situations here I would have loved to test, however because of how jestDOM works I'm not entirely sure it's possible

Specifically 
- focus on the error dialog
- moving via the buttons and re-focusing on the stepper item

Reply:
I spent half a day trying to figure out how to test this and event looped in Lex for a bit on this one. 

Tried using the waitFor logic, and ended up using the userEvent for many of these to make them work in the first place. Did some digging, I don't think it's worth diving into more. Happy if someone else wants to help me out and look at it a bit

Reply:
Do you think this would be easier to test with an E2E test?

Reply:
I think it would be possible. Not sure if we want to?